index.src.html

```diff
@@ -91,11 +92,10 @@
       text: creation URL
     urlPrefix: workers.html
       text: running a worker; url: run a worker
-    urlPrefix: dom.html
-      text: document referrer policy; url: concept-document-referrer-policy; for: Document
   type: interface
     urlPrefix: dom.html
       text: Document
+      text: referrer policy; url: concept-document-referrer-policy; for: Document
```

Comment:
When I do that, it gets confused about multiple definitions for 'referrer policy' and chooses to link all the `<a>referrer policy</a>`'s in the spec to the HTML Document definition. Is there any way to tell it that I want the others to link to this spec's referrer policy dfn? I think I tried to put something in the link-defaults section but it didn't work; I'll play with that again...